components/item/ItemPreviewCard.vue

```diff
@@ -54,7 +54,7 @@
 
     computed: {
       texts() {
-        if (this.variant === 'similar') return [];
+        if (this.variant === 'similar' || this.variant === 'explore') return [];
```

Comment:
I'd use a check to see if the variant is in an array, but similar will be removed after the new page becomes the default.